docs/reference/ml/anomaly-detection/apis/get-bucket.asciidoc

```diff
@@ -18,12 +18,11 @@ Retrieves {anomaly-job} results for one or more buckets.
 [[ml-get-bucket-prereqs]]
 == {api-prereq-title}
 
-* If the {es} {security-features} are enabled, you must have `monitor_ml`,
-`monitor`, `manage_ml`, or `manage` cluster privileges to use this API. You also
-need `read` index privilege on the index that stores the results. The
-`machine_learning_admin` and `machine_learning_user` roles provide these
-privileges. For more information, see <<security-privileges>>,
-<<built-in-roles>>, and {ml-docs-setup-privileges}.
+Requires the following privileges:
+
+* cluster: `monitor_ml` (the `machine_learning_user` built-in role grants this 
+  privilege)
+* result index: `read`
```

Comment:
I think the built-in role also grants this (per https://www.elastic.co/guide/en/elasticsearch/reference/master/built-in-roles.html) and if possible I think we should be explicit about the indices you need access to:
```suggestion
* `.ml-anomalies*` indices: `read` (the `machine_learning_user` built-in role grants this privilege)
```

Reply:
You _don't_ need read access to the results index to call the `get buckets` API.  You only need read access to the results index if you want to bypass the official APIs for getting results and search the results index directly.

Reply:
Thank you for clarifying it!